packages/relay/src/lib/config/hbarSpendingPlanConfigService.ts

```diff
@@ -58,6 +58,14 @@ export class HbarSpendingPlanConfigService {
   private readonly SPENDING_PLANS_CONFIG_FILE: string =
     ConfigService.get('HBAR_SPENDING_PLANS_CONFIG_FILE') || 'spendingPlansConfig.json';
 
+  /**
+   * The environment variable that contains the spending plans configuration JSON.
+   *
+   * @type {string}
+   * @private
+   */
+  private readonly SPENDING_PLANS_CONFIG_JSON: string | undefined = process.env.HBAR_SPENDING_PLANS_CONFIG_JSON;
```

Comment:
Please, use `ConfigService` instead of `process.env` here, this should be the standard way of accessing env variables now:
```suggestion
  private readonly SPENDING_PLANS_CONFIG_JSON: string | undefined = ConfigService.get('HBAR_SPENDING_PLANS_CONFIG_JSON');
```